partner_subject_to_vat/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Partner Subject to VAT",
+    "summary": """
```

Comment:
This should go on one line. And in a simple pair of double quotes, instead of triple.